docs/download.md

```diff
@@ -31,19 +31,22 @@ The plugin is tested on the following architectures:
 
 	NVIDIA Driver*: 470+
 
-	Python 3.6+, Scala 2.12, Java 8, Java 11, Java 17
+	Runtime: 
+		Scala 2.12
+		Python, Java Virtual Machine (JVM) compatible with your spark-version. 
+
+		* Check the Spark documentation for Python and Java version compatibility at `https://spark.apache.org/docs/{spark-version}/`. Please be aware that we do not currently support Spark builds with Scala 2.13.
```

Comment:
We strive to keep the line length under around 100 characters. 

Optionally could add  a link to a concrete Spark version: like the latest or to the earliest

Reply:
yes let's make it at most 100 characters here.

Reply:
My bad, I have updated the PR